src/python/pants/scm/git.py

```diff
@@ -175,7 +175,7 @@ def changed_files(self, from_commit=None, include_untracked=False, relative_to=N
                                              raise_type=Scm.LocalException)
       files.update(committed_changes.split())
     if include_untracked:
-      untracked_cmd = ['ls-files', '--other', '--exclude-standard'] + rel_suffix
+      untracked_cmd = ['ls-files', '--other', '--exclude-standard', '--full-path'] + rel_suffix
```

Comment:
Does the `diff --name-only` above need something similar?

Reply:
`--name-only` is a flag for `ls-files` only as far as I can tell.